---
layout: default
title: "Varify"
order: 1
---

# *Verify* with Varify.  

![check](https://github.com/nmferraro5/nmferraro5.github.io/blob/master/checkmark.png?raw=true)
<br>
#### Quickly discover clinically relevant variants using integrated annotations sources.

<br>Next generation sequencing technologies are allowing researchers to generate millions of genomic
variants routinely, and this growth of data provides opportunities to uncover information involving
allele frequencies and variant differences between various samples, patients, and cohorts.  This large
amount of data presents data management and use challenges that Varify exists to solve.  Varify is
an integrated data warehouse and analysis suite that enables a researcher to annotate variants one
sample at a time. Varify was developed in colloboration with the NHGRI Clinical Sequencing Exploratory
Research program, and CHOP's clinical laboratories. It was built using
[Harvest](http://harvest.research.chop.edu/), an open-source, web-based biomedical application
development framework.  Varify's data warehouse was deployed on the open-source PostgreSQL relational
database.  Unique variants are combined from all samples into one table.  The report for each sample
includes a variety of annotations, and the user has the ability to sort his or her results according
to various filters.  Navigate through this site using the menu on the left to discover more details on
Varify's capabilities and how to access this tool.




