---
layout: default
title: "Annotations and Features"
category: ref
date: 2013-07-12
order: 2
---

### Annotations and Features

<b>Deleterious Probabilities</b>

SIFT
PolyPhen2

<b>Allele Frequency Data</b>

1000G, combined allele frequency, combined allele count
EVS, combined allele frequency, combined allele count

<b>Genes</b>

approved HGNC gene symbols, names, and synonyms
transcripts and exons from RefSeq that can be mapped to HGNC genes

<b>Gene Lists</b>

the single gene list will be in there as well

<b>Disease and Phenotype</b>

diseases loaded from HGMD linked at the gene level and variant level
early stages of phenotype data loaded from NCBO linked to gene

<b>Variant Effects</b>

SNPEff variant effects and impact

<b>PubMed IDs</b>

derived from HGMD and linked to genes and variants

<b>Project-level hierarchy</b>

all samples are tracked via the following hierarchy (this enables a permissions system): PROJECT → COHORT → SAMPLE → RESULTS

<b>Samples</b>

Standard VCF fields - genotype, quality, depth of coverage, etc.

<b>Analysis Workflow</b>

the first step is to select a sample that is of interest
a step-by-step dialog taking the user through most of the above listed annotations with proper defaults

<b>HGMD SNV-Phenotype Associations</b>

The first of ten mutation-disease association data tables provided by HGMD has been loaded. This includes phenotypes/disease associated with point mutations (SNVs). This introduces:

70,000+ variant-disease associations
43,000+ gene-disease associations
5100+ new phenotypes (in addition to HPO's 5200)

<b>Presence in HGMD</b>

Includes a filter under the Phenotypes tab that enables 
filtering variants based on their presence in HGMD. In addition, if there are known HGMD 
IDs associated with a variant, they are now visible in the Summary section of the variant result 
view.

<b>Chromosome and Position</b>

Samples can be filtered by chromosome and/or position in the analysis view. 


