---
layout: default
title: "Guide Workflow"
category: ref
date: 2013-07-12
order: 2
---

### Guide Workflow

Varify's guided analysis workflow is two phase. The first step is to reduce the number of variants 
down to manageable size by applying filters that remove variants that are predicted to have very 
low or no impact on gene (and thus protein) function. The second step is to extract a slice of 
variants necessary to answer the question at hand.

The following is composed of the variant reduction steps.

<b>Common Variants</b> - compares variants against the 1000 Genomes and Exome Variant Server (EVS) data 
sets and removes the most common variants across the population<br>
<b>Deleterious Predictions</b> - uses the SIFT and PolyPhen2 scores to remove predicted benign 
variants<br>
<b>Altered Gene Function</b> - the effect a variant has on a gene (and downstream) is predicted via 
the SNPEff tool. Synonymous effects can be removed from the set as they have low or no impact on the 
gene function.<br>

![Analysis](https://github.com/nmferraro5/nmferraro5.github.io/blob/master/_posts/analysis.jpg?raw=true)

At this point, the number of variants should be between 100-1800 variants per sample (for exome 
analysis) depending on the strictness of the filters that were applied. The next phase is to extract 
a slice of variants that are relevant to the analysis. The following additional filters can applied 
to achieve this:

<b>Target Genes</b> - a list of genes can be specified which is appropriate for targeting a disease 
if the genes are known.<br>
<b>Gene Function</b> - if the gene function has been affected in a particular way that is known to 
cause downstream effects, the variants can be reduced based on their effect (this is similar to the 
above reduction filter)<br>
<b>Phenotype/Disease</b> - if a particular phenotype is of interest, the variant set can be reduced 
down to the known phenotype/disease-causing variants<br>

![Sort](https://github.com/nmferraro5/nmferraro5.github.io/blob/master/_posts/sort.jpg?raw=true)

Applying these additional filters should reduce the set down to at most a couple hundred variants. 
This is a reasonable number to wade through for finding novel variants for the samples of interest.

After applying these filters, individual variants can be viewed and the information available for that variant is displayed.

![Variants](https://github.com/nmferraro5/nmferraro5.github.io/blob/master/_posts/variant_details.jpg?raw=true)

Under the knowledge capture tab, the user can enter information about that variant to be stored in the database.

![Knowledge](https://github.com/nmferraro5/nmferraro5.github.io/blob/master/_posts/knowledge_capture.jpg?raw=true)

